hatchet/tests/conftest.py

```diff
@@ -347,6 +347,176 @@ def mock_graph_literal():
     return graph_dict
 
 
+@pytest.fixture
+def mock_graph_literal_inc_time_as_foo():
+    """Same as mock_graph_literal, but the "time (inc)"
+    metric is renamed to "foo"
```

Comment:
Since `foo` is one of the node names in this graph, can we change the metric name `foo` to a common metric name, like `line`?